sdk/lib/atl/atlimage.h

```diff
@@ -710,7 +710,7 @@ class CImage
         excludeDefaultSave  = excludeIcon | excludeEMF | excludeWMF
     };
 
-protected:
+private:
```

Comment:
Are those changes compatible with MS ATL?

Reply:
ATL uses `private` instead of `protected` here.